Tests/Functional/Controller/ConnectTest.php

```diff
@@ -70,24 +56,40 @@ public function test_exporting_a_product_with_variants()
         $this->importFixturesFileOnce(__DIR__ . '/_fixtures/config_fixes.sql');
         $this->importFixturesFileOnce(__DIR__ . '/_fixtures/connect_items.sql');
 
-        $client->request('POST', 'backend/Connect/insertOrUpdateProduct', ['sourceIds' => ['2','2-123','2-124']]);
+        $sourceIds = ['2','2-123','2-124'];
+
+        $client->request('POST', 'backend/Connect/insertOrUpdateProduct', ['sourceIds' => $sourceIds]);
         $responseData = $this->handleJsonResponse($client->getResponse());
 
         $this->assertTrue($responseData['success']);
+
+        $exportedItems = self::getDbalConnection()->fetchAll(
+            "SELECT export_status FROM s_plugin_connect_items WHERE source_id in (?, ?, ?) AND export_status = 'update' ",
```

Comment:
generally this works
but you can use
'SQL with WHERE x IN (?)',
[sourceIds],
\Doctrine\DBAL\Connection::PARAM_INT_ARRAY

this will work for arrays with any length even if you don't know the length beforehand
Also please capitalize IN it is an mysql keyword